_rules/image-filename-as-accessible-name-9eb3f6.md

```diff
@@ -61,45 +68,88 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
 
-This `img` element has an [accessible name][] equivalent to the filename. The [accessible name][] accurately describes the image.
+This `img` element has an [accessible name][] equivalent to the filename (ignoring letter casing). The [accessible name][] accurately describes the image.
 
 ```html
 <html lang="en">
-	<img src="https://www.w3.org/WAI/demos/bad/img/w3c" alt="w3c" />
+	<img src="/test-assets/image-filename-as-accessible-name-9eb3f6/nyhavn" alt="Nyhavn" />
```

Comment:
Does this actually work? I'm concerned the file you're referencing here isn't going to get served with the right HTTP headers, and so this example won't work. @jkodu any thoughts on how we might make this work?

Reply:
My tests indicate this works in Chrome, Edge, and Firefox (the image is displayed correctly).
Probably thanks to magic number in the file.